application/server/configuration.go

```diff
@@ -145,6 +145,7 @@ func writeSettings(c *config.Config, settings lsp.Settings, initialize bool) {
 	updateRuntimeInfo(c, settings)
 	updateAutoScan(c, settings)
 	updateSnykLearnCodeActions(c, settings)
+	updateSnykOSSQuickFixCodeActions(settings)
```

Comment:
Could you pass the config into the func please, instead of accessing the config via CurrentConfig? That way injection and parallelism in testing is a bit easier.